docs/en/styling-and-theming/basic-usage.md

```diff
@@ -0,0 +1,177 @@
+# Basic Usage
+
+## Styling a widget
+
+For a given application (this will be relevant in later examples when specifying [widget theme keys](./supplemental.md#widget-theme-keys)):
+
+> package.json
+
+```json
+{
+	"name": "my-app"
+}
+```
+
+Defining [per-widget CSS modules](./supplemental.md#structural-widget-styling):
+
+> src/styles/MyWidget.m.css
+
+```css
+.root {
+	font-family: sans-serif;
+}
+```
+
+> src/widgets/MyWidget.tsx
+
+```tsx
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+
+import * as css from '../styles/MyWidget.m.css';
+
+export default class MyWidget extends WidgetBase {
+	protected render() {
+		return <div classes={[css.root]}>My Widget</div>;
+	}
+}
+```
+
+## Making a widget themeable
+
+Extending `ThemedMixin`, applying the `@theme()` decorator to a widget's imported CSS modules, and wrapping CSS class names in calls to `this.theme()` when rendering to allow a [widget's default styles to be overridden by a theme](./supplemental.md#making-themeable-widgets):
```

Comment:
This sentence is pretty dense and can be difficult to follow. Might be better as a list?
* extend `ThemedMixin`
* apply `@theme()` decorator
* use `this.theme()`